### PR TITLE
fix crashing if music is scanning and application gets stopped

### PR DESCRIPTION
**Pull request by Rechi**

### CODE REVIEW

File: xbmc/music/infoscanner/MusicInfoScanner.cpp [hunk 1]
```diff
@@ -393,12 +393,12 @@ bool CMusicInfoScanner::IsScanning()
   return m_bRunning;
 }
 
-void CMusicInfoScanner::Stop()
+void CMusicInfoScanner::Stop(bool wait /*= false*/)
```
Comment (Montellese) [inline xbmc/music/infoscanner/MusicInfoScanner.cpp]: Please add a space between `/*` and `=` and after `false` for better readability.

**Comment by Montellese** [comment]
Looks sane IMO.

**Comment by Rechi** [comment]
@Montellese updated

**Comment by MartijnKaijser** [comment]
jenkins build this please
@Montellese backport?

**Comment by Montellese** [comment]
Maybe but it could go into a point release as well.

**Comment by MartijnKaijser** [comment]
well RC1 only to be released so that gives better testing ground than doing it in a point release imo. Up to you of course

**Comment by arnova** [comment]
I think this is pretty safe for backport. Fix is rather straightforward.

**Pull request merged**